app/pages/editorg/template.njk

```diff
@@ -37,8 +37,8 @@
           checked: agreementSigned,
           items: [{
             id: "catalogue-agreement-checkbox",
-            name: "catalogueagreementsigned",
-            value: agreementSigned,
+            name: "catalogueAgreementSigned",
+            value: 'true',
```

Comment:
do we need this? also it's always hardcoded to `'true'`

Reply:
this is the value given to the checkbox when it is checked - if it is left blank it defaults to '' which is falsey

Reply:
But looks like we are always setting it to `'true'` though. Regardless of whether it is checked or not

Reply:
this value comes through on the body